skyportal/handlers/api/followup_request.py

```diff
@@ -441,6 +441,8 @@ def get(self, followup_request_id=None):
                 obj_subquery, FollowupRequest.obj_id == obj_subquery.c.id
             )
         if instrumentID:
+            # allocation query required as only way to reach
+            # instrument_id
```

Comment:
I don't get it. Why not just query all followup requests with this `instrumentID`? Is the allocations subquery supposed to filter only followup requests that have an allocation visible to the user or something like that?

Reply:
followup_requests don't have an instrument_id, they have an allocation_id

Reply:
OK that's really surprising. Can you add that to the comment? I'm sure someone like me would run into it at some point and would take a while to figure this one out.